build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath 'com.android.tools.build:gradle:3.6.1'
```

Comment:
@nikhil-roy I have updated the gradle verison. It is Ok ?

Reply:
its fine